<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Proxy</class>
 <widget class="QDialog" name="Proxy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure proxy</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="1">
    <widget class="QCheckBox" name="checkBoxRememberSettings">
     <property name="text">
      <string>Remember this setting for next time</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelHostname">
     <property name="text">
      <string>Hostname</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditPort">
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelPassword">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEditPassword">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelUsername">
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelPort">
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="comboBoxProxyType"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditUsername"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelType">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditHostname"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="labelRestartWarning">
     <property name="text">
      <string>[[proxy-restart]]</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelWarningIcon">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../huggle_res/pictures.qrc">:/huggle/pictures/Resources/warning_icon_micro.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditHostname</tabstop>
  <tabstop>lineEditPort</tabstop>
  <tabstop>lineEditUsername</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>comboBoxProxyType</tabstop>
  <tabstop>checkBoxRememberSettings</tabstop>
 </tabstops>
 <resources>
  <include location="../huggle_res/pictures.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Proxy</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Proxy</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
